### PR TITLE
added basic book controller implementation

### PR DESCRIPTION
**Pull request by Tarabarnik**

### CODE REVIEW

File: src/main/java/library/app/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/book")
+public class BookController {
+
+    @GetMapping
+    public String getAllBooks() {
+        return "allBooks";
```
Comment (Sofasmile) [inline src/main/java/library/app/controller/BookController.java:22]: add implementation for methods

File: src/main/webapp/WEB-INF/views/allBooks.jsp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%--
```
Comment (Sofasmile) [inline src/main/webapp/WEB-INF/views/allBooks.jsp]: remove commented code from jsp

File: src/main/java/library/app/config/WebConfig.java [hunk 3]
```diff
@@ -9,11 +12,29 @@
...
 
+import javax.annotation.PostConstruct;
+
 @Configuration
 @EnableWebMvc
 @ComponentScan("library.app.controller")
 public class WebConfig extends WebMvcConfigurerAdapter {
 
+    @Autowired
+    private BookService bookService;
```
Comment (Sofasmile) [inline src/main/java/library/app/config/WebConfig.java]: better create separate class for injecting

File: src/main/java/library/app/controller/BookController.java [hunk 4]
```diff
@@ -1,20 +1,31 @@
...
     @GetMapping
-    public String getAllBooks() {
+    public String getAllBooks(ModelMap model) {
+        model.put("books", bookDao.listBooks());
         return "allBooks";
     }
 
     @GetMapping("/info")
-    public String bookInfo() {
+    public String bookInfo(ModelMap model, @RequestParam Long id) {
```
Comment (Sofasmile) [inline src/main/java/library/app/controller/BookController.java:26]: in next homework try to use `PathVariable`?

File: src/main/java/library/app/controller/BookController.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@RequestMapping("/book")
+public class BookController {
+
+    @Autowired
+    private BookDao bookDao;
```
Comment (Sofasmile) [inline src/main/java/library/app/controller/BookController.java]: can we use bookDao in controller???

File: src/main/java/library/app/controller/BookController.java (hunk 4 above)
Comment (Tarabarnik) [reply src/main/java/library/app/controller/BookController.java:26]: OK. But Bogdan told today, we'd better use RequestParam for id

Comment (Sofasmile) [reply src/main/java/library/app/controller/BookController.java:26]: copy from slack:
book info - use @ PathVariable

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/library/app/controller/BookController.java (hunk 4 above)
Comment (Tarabarnik) [reply src/main/java/library/app/controller/BookController.java:26]: Sorry, mixed up :)

**Pull request merged**